zproject/computed_settings.py

```diff
@@ -442,10 +442,7 @@
 S3_KEY = get_secret("s3_key")
 S3_SECRET_KEY = get_secret("s3_secret_key")
 
-# GCM tokens are IP-whitelisted; if we deploy to additional
-# servers you will need to explicitly add their IPs here:
-# https://cloud.google.com/console/project/apps~zulip-android/apiui/credential
-ANDROID_GCM_API_KEY = get_secret("android_gcm_api_key")
+# TODO: What's the correct version of the old comment?
```

Comment:
As discussed, this comment have not even be true perhaps? Either way, flagging this to decide whether to just leave this deleted or if we want to document some up-to-date equivalent of this somewhere.

Reply:
I think leaving it deleted is correct; this sort of detail belongs in other documentation anyway.